### DISCUSSION TITLE
How can I resolve `ErrOptimisticLock` error?

### DISCUSSION THREAD

**Discussion by hyeyoung-leee** [goharbor/harbor#19258] [Q&A]
Hi, 
How can I resolve the error with "the object has been modified; please apply your changes to the latest version and try again" message?
And what is the cause of the error?

This error occurs in `updateUsageWithRetry` function in `/src/controller/quota/controller.go`.

**Comment by stonezdj** [comment]
see: https://github.com/goharbor/perf/wiki/Quota-Update
